apps/homescreen/js/appsmetadata.js

```diff
@@ -47,12 +52,12 @@
           }
 
           if (typeof entry.order !== 'undefined') {
-            txn.objectStore(DB_ORDER_STORE).
-              put({ id: entry.id, order: entry.order });
+            txn.objectStore(DB_ORDER_STORE)
+              .put({ id: entry.id, order: entry.order });
```

Comment:
nit: personally I prefer to have the operator at the end of the broken line rather than the beginning of the new line (I find it better for readability).